src/NZSL/Ast/AstSerializer.cpp

```diff
@@ -83,6 +83,10 @@ namespace nzsl::Ast
 		Container(node.parameters);
 		for (auto& param : node.parameters)
 			Node(param);
+
+		Container(node.parametersSemantic);
```

Comment:
For retrocompatibility we should check the binary version of the deserializer (IsVersionGreaterThan)

Reply:
Fixed